group_project_v2/utils.py

```diff
@@ -355,3 +360,28 @@ def is_absolute(url):
     Returns a boolean value indicating if given `url` is absolute or not.
     """
     return bool(urlparse.urlparse(url).netloc)
+
+
+def make_s3_link_temporary(group_id, file_sha1, file_name, file_url):
+
```

Comment:
Can you add a docstring to this function explaining why it's used and how it works?